### PR TITLE
Translation integration section in French

### PR DESCRIPTION
**Pull request by iGitScor**
**What**: Translation "integration" section in french

**Why**: To be more accessible to French speakers

**How**: French is my mother tongue 🇫🇷

**Issue**: https://github.com/kentcdodds/glamorous-website/issues/239

### CODE REVIEW

**Comment by kentcdodds** [comment]
Super! Looks good to me! @iGitScor could you get one or more french speakers to review this? Maybe @ mention some of the reviewers of your past PRs?

**Comment by iGitScor** [comment]
Sorry I didn't notice your comment. I'm going to try to find help for french content reviewers in addition to previous reviewers

**Comment by kentcdodds** [comment]
cc @forresst, @Macxim, @etiennetalbot, @bulby97, @Philmod, @xtuc, @garlab, and @Tom-Bonnike

Thanks for any help you can give :)

**Comment by xtuc** [comment]
French guys are usually not used to read and talk to english peoples. Making a french accessible website is awesome, thanks!

**Comment by iGitScor** [comment]
I totally agree with @xtuc but junior french devs like students are not all comfortable with English and could learn interesting concepts with some translated documentation

**Comment by xtuc** [comment]
@iGitScor yes.

On the website, there is 'initial' written in the language menu, is this normal ?

screenshot from 2017-07-25 08-46-55

**Comment by iGitScor** [comment]
Nope, if you could [submit an issue](https://github.com/kentcdodds/glamorous-website/issues/new), it would be nice. It's not a translation issue but a style issue.

I'll may try to fix later it if I can.

👏 Thanks

**Comment by kentcdodds** [comment]
Yeah, I've fixed that already. Don't worry about it 😅

File: pages/integrations/content/fr/glamor.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+  '100%': { transform: `scale(0.9)` }
+})
+
+// Création d'un composant utilisant le style défini auparavant
+const AnimatedDiv = glamorous.div({
+  fontSize: 50,
+  width: '100%',
+  textAlign: 'center',
+
+  // Animer le div avec les keyframes
```
Comment (Tom-Bonnike) [inline pages/integrations/content/fr/glamor.md]: Nitpick: Don’t we usually say `la` div? I know there’s probably not a specific rule but… haha.

File: pages/integrations/content/fr/recompose.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
 ---
 title: '[`recompose`](https://github.com/acdlite/recompose)'
+contributors:
+  - kentcdodds
+  - igitscor
 ---
+
+`glamorous` crée des composants simples, si vous souhaitez complexifier
```
Comment (Tom-Bonnike) [inline pages/integrations/content/fr/recompose.md]: Nitpick: Maybe split the sentence? The comma use seems strange to me here.

`glamorous` crée des composants simples. Si vous souhaitez complexifier…
Or add:
`glamorous` crée des composants simples mais si vous souhaitez complexifier…

File: pages/integrations/content/fr/glamor.md (hunk 1 above)
Comment (iGitScor) [inline pages/integrations/content/fr/glamor.md]: I use `le` because `section`/`block`/`element` are male words

Comment (Tom-Bonnike) [inline pages/integrations/content/fr/glamor.md]: Ok I guessed so! Personal preference, whatever :)

File: pages/integrations/content/fr/recompose.md (hunk 2 above)
Comment (iGitScor) [inline pages/integrations/content/fr/recompose.md]: 👍

**Comment by kentcdodds** [comment]
Great! Thanks everyone! 👏

**Pull request merged**